cpp/examples/ode_secir.cpp

```diff
@@ -84,7 +86,7 @@ int main()
     bool print_to_terminal = true;
 
     if (print_to_terminal) {
-        char vars[] = {'S', 'E', 'C', 'I', 'H', 'U', 'R', 'D'};
+        char vars[] = {'S', 'E', 'C', 'C', 'I', 'I', 'H', 'U', 'R', 'D'};
         printf("\n # t");
         for (size_t k = 0; k < (size_t)mio::osecir::InfectionState::Count; k++) {
             printf(" %c", vars[k]);

```

Comment:
```suggestion
        std::vector<std::string> vars = {"S", "E", "C", "C_confirmed", "I", "I_confirmed", "H", "U", "R", "D"};
        printf("\n # t");
        for (size_t k = 0; k < (size_t)mio::osecir::InfectionState::Count; k++) {
            printf(" %s", vars[k].c_str());
        }
```
I think this is confusing. We should not use the same char for two compartments. However, we have to change to string. Maybe theres an better solution.
Additionally, this part will be redesigned once PR #511 is merge since we added a function for printing as member function of timeseries